Dockerfile

```diff
@@ -1,28 +1,32 @@
 # Client
-FROM public.ecr.aws/bitnami/node:14.17.3 AS client
+FROM registry.access.redhat.com/ubi8/nodejs-14:1 AS client
```

Comment:
Does using this image mean we don't need to download the CLI in the build step?

Reply:
nope CLI is required in pipeline. But that is already preinstalled in `ubuntu-latest` runner.